frontend/src/screens/wallet/Send.tsx

```diff
@@ -16,98 +10,123 @@ import {
   AlertTitle,
 } from "src/components/ui/alert.tsx";
 import { Button } from "src/components/ui/button";
-import {
-  Card,
-  CardContent,
-  CardFooter,
-  CardHeader,
-  CardTitle,
-} from "src/components/ui/card";
 import { Input } from "src/components/ui/input";
 import { Label } from "src/components/ui/label";
-import { LoadingButton } from "src/components/ui/loading-button";
 import { useToast } from "src/components/ui/use-toast";
+import BalanceCard from "src/components/wallet/BalanceCard";
+import ConfirmPayment from "src/components/wallet/ConfirmPayment";
+import LnurlPay from "src/components/wallet/LnurlPay";
 import { useBalances } from "src/hooks/useBalances";
 import { useChannels } from "src/hooks/useChannels";
 
 import { useInfo } from "src/hooks/useInfo";
-import { copyToClipboard } from "src/lib/clipboard";
-import { PayInvoiceResponse } from "src/types";
-import { request } from "src/utils/request";
+
+// email regex: https://emailregex.com/
+// modified to allow _ in subdomains
+const LIGHTNING_ADDRESS_REGEX =
```

Comment:
I think this is in a few projects now, I wonder if it should be moved to something like `lightning-tools`